plugins/ccxtExchange_test.go

```diff
@@ -471,11 +473,17 @@ func TestGetOrderConstraints_Ccxt_Precision(t *testing.T) {
 			wantPricePrecision: 8,
 			wantVolPrecision:   0,
 		},
+		{
+			exchangeName:       "coinbasepro",
+			pair:               &model.TradingPair{Base: model.XLM, Quote: model.USD},
+			wantPricePrecision: 6,
+			wantVolPrecision:   0,
```

Comment:
can you share the link where to find precision values for coinbasepro here? may be useful to put this as a comment inside each of these test cases
(don't worry about the kraken/binance cases, I'll add these links in)